erts/emulator/beam/beam_emu.c

```diff
@@ -1798,7 +1815,8 @@ void process_main(Eterm * x_reg_array, FloatDef* f_reg_array)
 		StoreBifResult(3, result);
 	    }
 	}
-     goto badarg;
+	c_p->freason = BADARG;
+	BIF_ERROR_ARITY_2(make_small(Arg(2)), fast_element_tuple, BIF_element_2);
```

Comment:
Please review this particular entry. I am not a 100% sure making it small is the way to go.